components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/pom.xml

```diff
@@ -170,6 +170,13 @@
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
                 <version>${jacoco.version}</version>
+                <configuration>
+                    <excludes>
+                        <exclude>**/TenantMgtAdminServiceClient*</exclude>
+                        <exclude>**/UserRegistrationAdminServiceClient*</exclude>
+                        <exclude>**/TenantDataManager*</exclude>
```

Comment:
Better to mention as a comment why these classes are excluded.

Reply:
+1.
Done!